docs/release-notes/dracut_visible_warnings.rst

```diff
@@ -0,0 +1,11 @@
+:Type: Initial RAM disk (initrd)
+:Summary: Make installer critical warnings in Dracut/initrd more visible (#1983098)
```

Comment:
I think swapping the adjectives might work better: `...critical installer...` However the adjective ordering rules don't fit too well here, so feel free to disagree.

Reply:
Fixed. Thanks for suggestion.